airflow/cli/commands/info_command.py

```diff
@@ -279,6 +285,13 @@ def __str__(self):
         )
 
 
+class ProvidersInfo:
+    """providers information"""
+
+    def __str__(self):
+        return tabulate_providers(ProvidersManager().providers.values(), "grid")
```

Comment:
Do we need descriptions for each package here? I think the provider name would be enough, although I personally would probably only use the package name and version. https://github.com/apache/airflow/blob/master/airflow/provider.yaml.schema.json#L9-L12

```
Apache Airflow [2.0.0b3]

Platform: [Mac OS, x86_64] uname_result(system='Darwin', node='Kamils-MacBook-Pro.local', release='20.1.0', version='Darwin Kernel Version 20.1.0: Sat Oct 31 00:07:11 PDT 2020; root:xnu-7195.50.7~2/RELEASE_X86_64', machine='x86_64', processor='i386')
Locale: ('en_US', 'UTF-8')
Python Version: [3.6.8 (v3.6.8:3c6b436a57, Dec 24 2018, 02:04:31)  [GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.57)]]
Python Location: [${HOME}/.virtualenvs/airflow/bin/python3.6]

git: [git version 2.28.0]
ssh: [OpenSSH_8.1p1, LibreSSL 2.7.3]
kubectl: [Client Version: v1.18.8]
gcloud: [Google Cloud SDK 307.0.0]
cloud_sql_proxy: [NOT AVAILABLE]
mysql: [mysql  Ver 8.0.22 for osx10.15 on x86_64 (Homebrew)]
sqlite3: [3.32.3 2020-06-18 14:16:19 02c344aceaea0d177dd42e62c8541e3cab4a26c757ba33b3a31a43ccc7d4aapl]
psql: [psql (PostgreSQL) 13.0]

Airflow Home: [${HOME}/airflow]
System PATH: [${HOME}/.virtualenvs/airflow/bin:${HOME}/.krew/bin:${HOME}/tools/google-cloud-sdk/bin:${HOME}/.krew/bin:/usr/local/opt/ncurses/bin:/usr/local/opt/gettext/bin:/Library/Frameworks/Python.framework/Versions/3.7/bin:/Library/Frameworks/Python.framework/Versions/3.7/bin:${HOME}/bin/:/usr/local/opt/gnu-getopt/bin:/Library/Frameworks/Python.framework/Versions/3.7/bin:/Library/Frameworks/Python.framework/Versions/3.6/bin:/Library/Frameworks/Python.framework/Versions/2.7/bin:/usr/local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/VMware Fusion.app/Contents/Public:/usr/local/MacGPG2/bin:/Applications/Wireshark.app/Contents/MacOS]
Python PATH: [${HOME}/.virtualenvs/airflow/bin:${HOME}/.virtualenvs/airflow/lib/python36.zip:${HOME}/.virtualenvs/airflow/lib/python3.6:${HOME}/.virtualenvs/airflow/lib/python3.6/lib-dynload:/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6:${HOME}/.virtualenvs/airflow/lib/python3.6/site-packages:${HOME}/devel/google-airflow/airflow:${HOME}/airflow/dags:${HOME}/airflow/config:${HOME}/airflow/plugins]
airflow on PATH: [True]

Executor: [SequentialExecutor]
Task Logging Handlers: [airflow.utils.log.file_task_handler.FileTaskHandler]
SQL Alchemy Conn: [sqlite://Users/kamilbregula/airflow/airflow.db]
DAGS Folder: [${HOME}/airflow/dags]
Plugins Folder: [${HOME}/airflow/plugins]
Base Log Folder: [${HOME}/airflow/logs]

+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| Provider name                             | Description                                                                                            | Version   |
+===========================================+========================================================================================================+===========+
| apache-airflow-providers-amazon           | Amazon integration (including `Amazon Web Services (AWS) <https://aws.amazon.com/>`__).                | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-apache-cassandra | `Apache Cassandra <http://cassandra.apache.org/>`__.                                                   | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-apache-druid     | `Apache Druid <https://druid.apache.org/>`__.                                                          | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-apache-hdfs      | `Hadoop Distributed File System (HDFS) <https://hadoop.apache.org/docs/r1.2.1/hdfs_design.html>`__     | 1.0.0b2   |
|                                           | and `WebHDFS <https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-hdfs/WebHDFS.html>`__. |           |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-apache-hive      | `Apache Hive <https://hive.apache.org/>`__                                                             | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-apache-kylin     | `Apache Kylin <https://kylin.apache.org/>`__                                                           | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-apache-livy      | `Apache Livy <https://livy.apache.org/>`__                                                             | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-apache-pig       | `Apache Pig <https://pig.apache.org/>`__                                                               | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-apache-pinot     | `Apache Pinot <https://pinot.apache.org/>`__                                                           | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-apache-spark     | `Apache Spark <https://spark.apache.org/>`__                                                           | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-apache-sqoop     | `Apache Sqoop <https://sqoop.apache.org/>`__                                                           | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-celery           | `Celery <http://www.celeryproject.org/>`__                                                             | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-cloudant         | `IBM Cloudant <https://www.ibm.com/cloud/cloudant>`__                                                  | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-cncf-kubernetes  | `Kubernetes <https://kubernetes.io/>`__                                                                | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-databricks       | `Databricks <https://databricks.com/>`__                                                               | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-datadog          | `Datadog <https://www.datadoghq.com/>`__                                                               | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-dingding         | `Dingding <https://oapi.dingtalk.com/>`__                                                              | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-discord          | `Discord <https://discordapp.com/>`__                                                                  | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-docker           | `Docker <https://docs.docker.com/install/>`__                                                          | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-elasticsearch    | `Elasticsearch <https://https//www.elastic.co/elasticsearch>`__                                        | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-exasol           | `Exasol <https://docs.exasol.com/home.htm>`__                                                          | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-facebook         | `Facebook Ads <http://business.facebook.com/>`__                                                       | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-ftp              | `File Transfer Protocol (FTP) <https://tools.ietf.org/html/rfc114>`__                                  | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-google           | Google services including:                                                                             | 1.0.0b2   |
|                                           |                                                                                                        |           |
|                                           |   - `Google Ads <https://ads.google.com/>`__                                                           |           |
|                                           |   - `Google Cloud (GCP) <https://cloud.google.com/>`__                                                 |           |
|                                           |   - `Google Firebase <https://firebase.google.com/>`__                                                 |           |
|                                           |   - `Google Marketing Platform <https://marketingplatform.google.com/>`__                              |           |
|                                           |   - `Google Workspace <https://workspace.google.pl/>`__ (formerly Google Suite)                        |           |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-grpc             | `gRPC <https://grpc.io/>`__                                                                            | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-hashicorp        | Hashicorp including `Hashicorp Vault <https://www.vaultproject.io/>`__                                 | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-http             | `Hypertext Transfer Protocol (HTTP) <https://www.w3.org/Protocols/>`__                                 | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-imap             | `Internet Message Access Protocol (IMAP) <https://tools.ietf.org/html/rfc3501>`__                      | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-jdbc             | `Java Database Connectivity (JDBC) <https://docs.oracle.com/javase/8/docs/technotes/guides/jdbc/>`__   | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-jenkins          | `Jenkins <https://jenkins.io/>`__                                                                      | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-jira             | `Atlassian Jira <https://www.atlassian.com/>`__                                                        | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-microsoft-azure  | `Microsoft Azure <https://azure.microsoft.com/>`__                                                     | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-microsoft-mssql  | `Microsoft SQL Server (MSSQL) <https://www.microsoft.com/en-us/sql-server/sql-server-downloads>`__     | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-microsoft-winrm  | `Windows Remote Management (WinRM) <https://docs.microsoft.com/en-us/windows/win32/winrm/portal>`__    | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-mongo            | `MongoDB <https://www.mongodb.com/what-is-mongodb>`__                                                  | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-mysql            | `MySQL <https://www.mysql.com/products/>`__                                                            | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-odbc             | `ODBC <https://github.com/mkleehammer/pyodbc/wiki>`__                                                  | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-openfaas         | `OpenFaaS <https://www.openfaas.com/>`__                                                               | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-opsgenie         | `Opsgenie <https://www.opsgenie.com/>`__                                                               | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-oracle           | `Oracle <https://www.oracle.com/en/database/>`__                                                       | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-pagerduty        | `Pagerduty <https://www.pagerduty.com/>`__                                                             | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-papermill        | `Papermill <https://github.com/nteract/papermill>`__                                                   | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-plexus           | `Plexus <https://plexus.corescientific.com/>`__                                                        | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-postgres         | `PostgreSQL <https://www.postgresql.org/>`__                                                           | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-presto           | `Presto <https://prestodb.github.io/>`__                                                               | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-qubole           | `Qubole <https://www.qubole.com/>`__                                                                   | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-redis            | `Redis <https://redis.io/>`__                                                                          | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-salesforce       | `Salesforce <https://www.salesforce.com/>`__                                                           | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-samba            | `Samba <https://www.samba.org/>`__                                                                     | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-segment          | `Segment <https://segment.com/>`__                                                                     | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-sendgrid         | `Sendgrid <https://sendgrid.com/>`__                                                                   | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-sftp             | `SSH File Transfer Protocol (SFTP) <https://tools.ietf.org/wg/secsh/draft-ietf-secsh-filexfer/>`__     | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-singularity      | `Singularity <https://sylabs.io/guides/latest/user-guide/>`__                                          | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-slack            | `Slack <https://slack.com/>`__                                                                         | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-snowflake        | `Snowflake <https://www.snowflake.com/>`__                                                             | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-sqlite           | `SQLite <https://www.sqlite.org/>`__                                                                   | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-ssh              | `Secure Shell (SSH) <https://tools.ietf.org/html/rfc4251>`__                                           | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-vertica          | `Vertica <https://www.vertica.com/>`__                                                                 | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-yandex           | Yandex including `Yandex.Cloud <https://cloud.yandex.com/>`__                                          | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+
| apache-airflow-providers-zendesk          | `Zendesk <https://www.zendesk.com/>`__                                                                 | 1.0.0b2   |
+-------------------------------------------+--------------------------------------------------------------------------------------------------------+-----------+

```

Reply:
You are right. I will simplify it

Reply:
![Screenshot from 2020-11-28 12-56-54](https://user-images.githubusercontent.com/595491/100514981-3c747100-3179-11eb-8c33-1042a55fd205.png)